cmip6_downscaling/workflows/ERA5/ERA5_transfer.py

```diff
@@ -16,40 +17,103 @@
 # Helper Functions -----------------------------------------------------------
 
 
-def zarr_is_complete(store, check=".zmetadata"):
-    """Return true if Zarr store is complete"""
+def zarr_is_complete(store: str, check: str = ".zmetadata") -> bool:
+    """Return true if Zarr store is complete
+
+    Parameters
+    ----------
+    store : str
+        store name
+    check : str, optional
+        check name, by default ".zmetadata"
+
+    Returns
+    -------
+    bool
+        True if zarr store is complete
+    """
     return check in store
 
 
-def map_tgt(tgt, connection_string):
-    """Uses fsspec to creating mapped object from target connection string"""
-    tgt_map = fsspec.get_mapper(tgt, connection_string=connection_string)
+def map_tgt(tgt: str) -> fsspec.FSMap:
+    """Uses fsspec to creating mapped object from target connection string
+
+    Parameters
+    ----------
+    tgt : str
+        path store
+
+    Returns
+    -------
+    fsspec.FSMap
+        fsspec mapped object
+    """
+    tgt_map = fsspec.get_mapper(tgt, connection_string)
     return tgt_map
 
 
-def copy_cleaned_data(xdf, tgt_map, overwrite=True):
-    """Copies xarray dataset to zarr store for given target"""
+def copy_cleaned_data(xdf: xr.Dataset, tgt_map: str, overwrite: bool = True):
```

Comment:
I think this could use a `-> None` return type but I'm not 100% sure. What have you seen folks use for implicit None returns?